docker/jenkins/jenkins.yaml

```diff
@@ -52,6 +53,42 @@ jobs:
           }
       }
 
+  - script: >
```

Comment:
I think this docker setup needs to be in the e-commerce repo and not here

Reply:
here https://github.com/xebialabs/e-commerce-microservice/blob/master/docker-compose/xebialabs/jenkins/jenkins.yaml